Website/src/utils/common.ts

```diff
@@ -1,8 +1,27 @@
-import {Inter} from "next/font/google";
+import { Inter } from "next/font/google";
+import { POITypeIcon, POITypeIconValues } from "@/utils/api";
 
-export const inter = Inter({ subsets: ['latin'] })
+export const inter = Inter({ subsets: ["latin"] });
 
 export const meta_info = {
-    title: 'RailTrail Admin Interface',
-    description: 'An administrative interface for the RailTrail rail vehicle management system.',
-}
\ No newline at end of file
+	title: "RailTrail Admin Interface",
+	description: "An administrative interface for the RailTrail rail vehicle management system."
+};
+
+export const POIIconImg: Readonly<Record<POITypeIcon, string>> = {
```

Comment:
I am still not quite sure if this file is the correct place for this definition, but I had to fit it somewhere...